src/node/install.sh

```diff
@@ -103,6 +103,13 @@ else
     apt-get -y install --no-install-recommends yarn
 fi
 
+# Install pnpm
```

Comment:
Can you bump the minor version? 
https://github.com/devcontainers/features/blob/c8590bef3dce55bfc110fd3abdf6cd28e7d7e93f/src/node/devcontainer-feature.json#L3